tests/upgrades/test_virtwho.py

```diff
@@ -78,6 +78,9 @@ def test_pre_create_virt_who_configuration(
             3. Report is sent to satellite.
             4. Virtual sku can be generated and attached.
         """
+        org = target_sat.api.Organization().search(query={'search': f'name={ORG_DATA["name"]}'})
+        if org:
+            target_sat.api.Organization(id=org[0].id).delete()
```

Comment:
```suggestion
            org.delete()
```

Reply:
> 
@Gauravtalreja1 Thank you very much for your good suggestion, This code has been replaced by dynamic name, so no need to delete the exist org.

Reply:
Yes, still instead of searching with API, we could use `org.delete()`